File: C++/Panagram Checking.cpp
class Solution
{
 public:
    bool checkPangram (string s) {
        unordered_set<char> st;
        for(int i=0;i<s.size();i++){
            if(s[i]>='a' and s[i]<='z'){
                int val = int(s[i])-32;
                st.insert(char(val));
            }
             if(s[i]>='A' and s[i]<='Z'){
                
               st.insert(s[i]);
            }
        }
        return (st.size()==26);
    }

};

File: JAVA/Panagram Checking.java
class Solution
{
    public static boolean checkPangram  (String s) {
        int[] arr = new int[26];
        
        for(int i=0;i<s.length();i++){
            char c = Character.toLowerCase(s.charAt(i));
            if((c - 'a') >= 0 && c - 'a' <= 26){
                arr[c -'a']++;
            }
        }
        for(int i=0;i<arr.length;i++){
            if(arr[i] == 0){
                return false;
            }
        }
        return true;
    }
}
